buzzmobile/simulation/sim_car_interface/src/sim_car_interface.cpp

```diff
@@ -35,13 +35,13 @@ double speed_measured_right = 0.0;
 double steer_set_point = 0.0;
 
 // TODO(sahit): change to rosparams
-constexpr double chassis_length = 0.33246;
-constexpr double chassis_width = 0.28732;
+constexpr double chassis_length = 1.8; 
+constexpr double chassis_width = 1.57;
```

Comment:
holy cow that's awful. Please be sure to track units in the comments (or var names, ie. `chassis_length_ft` just so that we don't have the problem of coming back in 6 months wondering what the hell 17.2 is.